lib/modules/manager/composer/extract.ts

```diff
@@ -197,5 +197,10 @@ export async function extractPackageFile(
     };
     res.managerData = managerData;
   }
+
+  if (composerJson.require?.php) {
```

Comment:
Strangely, `is.nonEmptyString()` didn't work because TS thought could still be `undefined`

Reply:
🤔